### DISCUSSION TITLE
add Korean Language

### DISCUSSION THREAD

**Discussion by bhsim7** [wtetsu/mouse-dictionary#94] [Ideas]
Hello
I'm developer in South Korea.

how about you guys think add korean language ?

Is it good?

I cannot speak japanese and I can speak english little....
and I'm new begginer github. but I'm studying github.

whatever if I join mouse-dictionary. I will trying to hard....
thank you.

**Comment by wtetsu** [comment]
What does “add korean language” mean?

* Is it adding a default dictionary?
* Is it support for reading functionality?

I have no knowledge of Korean, so I can't do it myself, but suggestions and pull requests are welcome.
